provisioning-telephony/config.yml

```diff
@@ -0,0 +1,72 @@
+---
+title: Provisioning Telephony
+description: This tutorial implements BYOC Cloud solution with a cloud-based carrier 
+categories:
+- Trunk
+- Telephony
+steps:
+- title: Introduction
+  content: |
+    This tutorial walks through the steps on how to implement BYOC Cloud in Genesys Cloud using [Purecloud Platform API](https://developer.mypurecloud.com/api/rest/client-libraries/index.html). In order for BYOC Cloud to work with a third-party carrier, the carrier must:
```

Comment:
Remove the host for links to dev center content: `[Purecloud Platform API](/api/rest/client-libraries/index.html)`. There's a few other instances of this. However, leave the resource center links as-is since they're on a different domain.

Reply:
If I understood this comment correctly do you mean to eliminate (https://developer.mypurecloud.com/) part since they are sitting on the same domain with the developer center tutorials?

Reply:
Yep. This will allow visitors using the regional TLDs to stay on their regional TLD when visiting other pages in the dev center. Note that the links do need to start with a slash (`/`).